frontend/lib/identification/otp_generator.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:otp/otp.dart';
+
+class OTPGenerator {
+  final String _base32TotpSecret;
+  final int _otpIntervalSeconds;
+  final int _otpLength;
+  final Algorithm _algorithm;
```

Comment:
I think these three values are fix all over our app. We could also just set their values as constants here.

Reply:
Did we document the values somewhere?

Reply:
@Schedulaar

Reply:
Yeah, they **should** be documented in the verification_protocol svg in the artefacts repo.
Currently, we have only documented that the algorithm should be SHA-256 and the otp code should have 6 digits (stored as an int). The otpIntervalSeconds should be 30 seconds, but this is not yet documented (I will do so shortly)

Reply:
Will do in #224